src/getters.js

```diff
@@ -57,8 +57,8 @@ export default function Getters (cosmosRESTURL) {
       return get(`/auth/accounts/${address}`)
         .then(res => {
           // HACK, hope for: https://github.com/cosmos/cosmos-sdk/issues/3885
-          let account = res.value || emptyAccount
-          if (res.type === `auth/DelayedVestingAccount`) {
+          let account = (res && res.value) || emptyAccount
```

Comment:
Works perfectly. What do you think about an early exit strategy:

```
if (!res) return emptyAccount
let account = res.value
```

Reply:
Works for me.